.gitignore

```diff
@@ -4,7 +4,6 @@ node_modules
 bundles
 coverage
 reports
-docs
```

Comment:
Docs needs to be kept out of github as it is auto-generated. Recommend you add this new static file to a different location